config/locales/en.yml

```
@@ -288,38 +290,56 @@ en:
     certificates_expiring: "%{number} certificates are expiring soon."
     find_out_more: Find out more about %{link} in the documentation.
     rotate_key_and_certificate: rotating your %{component} %{certificate} key and certificate
+    continue: Continue
+    'no': 'No'
+    'yes': 'Yes'
```

```suggestion
    yes: Yes
```
it doesn't need to be in quotes

---

In the comments at the top of the file it says `# The following keys must be escaped otherwise they will not be retrieved by the default I18n backend: # true, false, on, off, yes, no`

---

ah! didn't know no and yes too.. the value doesn't have to be though

---

Just tested it, the values do also need to be in quotes